src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/AuroraTopologyService.java

```diff
@@ -140,6 +141,16 @@ public void setClusterId(String clusterId) {
     this.clusterId = clusterId;
```

Comment:
this.clusterId = clusterId;
this.metricsContainer.setClusterId(this.clusterId);